README.md

```diff
@@ -1,2 +1,14 @@
-# Garcia_TP1
-Primera_entrega
+# **<u>Trabajo Práctico N° 1</u>**
+___
+Para este primer trabajo se tuvieron en cuenta los 3 primeros cápitulos del libro utilizado en el curso. Donde se abordaron temas como manejo entradas y salidas digitales, manejo del tiempo de ejecucion y retardos por software y señales analógicas. También otros puntos importantes como modularización y forma prolija de escribir el código fuente para que pueda ser más mantenible y reutilizable.
+
+Para poder llevar a cabo el objetivo planteado en la consigna del trabajo práctico, se trabajó con una libreria para el sensor de temperatura y humedad DHT11.
+
+<b>*Se pudo cumplir con el funcionamiento planteado, el cual era imprimir los valores de temperatura y humedad a través del monitor del puerto serie, comparar el valor de temperatura con un umbral establecido y si la temperatura era superior a dicho umbral se prendian las alarmas (LED indicador y buzzer) y el ventilador. En caso contrario, las alarmas y el ventilador se mantenian apagados y hay un LED verde indicador de condiciones normales. Además se contaba con dos botones, uno para la activación/desactivación del buzzer y otro para el ventilador.*</b>  
```

Comment:
No me queda claro por qué en este párrafo usas negritas y por qué usas itálicas. Sugiero no usar ninguna de las dos.

Reply:
Simplemente era para mostrar la conclusión y como modo de práctica de las herramientas del markdown, ya las he desactivado.

Reply:
Ok, veo que esto ya lo corregiste acá: https://github.com/manucho1586/Garcia_TP1/blob/Correcciones_2/README.md